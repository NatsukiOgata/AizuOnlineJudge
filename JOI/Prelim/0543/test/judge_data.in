9850
200
300
450
2600
2500
1000
500
520
980
8790
200
1500
800
750
600
1400
680
1280
380
8089
560
1236
236
136
745
348
965
234
2345
1830
200
180
120
110
80
120
200
120
320
9626
642
5800
154
643
258
125
86
23
257
0
