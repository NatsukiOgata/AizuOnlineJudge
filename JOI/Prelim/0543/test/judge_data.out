800
1200
1284
380
1638
